bindata/cloud-network-config-controller/managed/controller.yaml

```diff
@@ -203,6 +209,11 @@ spec:
         - name: cloud-token
           mountPath: /var/run/secrets/openshift/serviceaccount
           readOnly: true
+{{- if not (eq .AzureManagedClientID "")}}
+        - name: cncc-cert
+          mountPath: /mnt/certs
```

Comment:
Can we mount this under `/var/run/secrets/` instead? I am not really digging using `/mnt` here